python/cugraph/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+[build-system]
+
+requires = [
+    "wheel",
+    "setuptools",
+    "cython>=0.29,<0.30",
+    "scikit-build>=0.13.1",
+    "cmake>=3.20.1,!=3.23.0",
+    "ninja",
+]
+
+[tool.black]
+line-length = 79
+target-version = ["py36"]
+include = '\.py?$'
+exclude = '''
+/(
+    thirdparty |
+    \.eggs |
+    \.git |
+    \.hg |
+    \.mypy_cache |
+    \.tox |
+    \.venv |
+    _build |
+    buck-out |
+    build |
+    dist
+)/
```

Comment:
Leave this out unless cugraph is going to start linting with black.
```suggestion
```

Reply:
Addressed